pkg/ip/ip.go

```diff
@@ -804,5 +804,69 @@ func IPToPrefix(ip net.IP) *net.IPNet {
 
 // IsIPv4 returns true if the given IP is an IPv4
 func IsIPv4(ip net.IP) bool {
-	return ip.To4() != nil
+	return ip != nil && ip.To4() != nil
+}
+
+// IsIPv6 returns if netIP is IPv6.
+func IsIPv6(ip net.IP) bool {
+	return ip != nil && ip.To4() == nil
+}
+
+// GetSortedIPList returns a new net.IP slice in which the IPs are sorted.
+func GetSortedIPList(ipList []net.IP) []net.IP {
+	sortedIPList := make([]net.IP, len(ipList))
+	for i := 0; i < len(ipList); i++ {
+		sortedIPList[i] = ipList[i]
+	}
+
+	sort.Slice(sortedIPList, func(i, j int) bool {
+		return bytes.Compare(sortedIPList[i], sortedIPList[j]) < 0
+	})
+
+	return sortedIPList
+}
```

Comment:
Usually we should return an object like it is being done in here but golang has `sort.Strings()` where it doesn't return an ordered slice so maybe we can do the same to avoid memory allocations for the returned slice.